src/ng/interpolate.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
 
-var $interpolateMinErr = minErr('$interpolate');
+var $interpolateMinErr = angular.$interpolateMinErr = minErr('$interpolate');
+$interpolateMinErr.noconcat = function(text) {
+  return $interpolateMinErr('noconcat',
+      "Error while interpolating: {0}\nStrict Contextual Escaping disallows " +
+      "interpolations that concatenate multiple expressions when a trusted value is " +
+      "required.  See http://docs.angularjs.org/api/ng.$sce", text);
+};
+
+$interpolateMinErr.interr = function(text, err) {
+  return $interpolateMinErr('interr', "Can't interpolate: {0}\n{1}", text, err.toString());
+};
```

Comment:
Fixed it.  Not super happy with the fix.  It adds a stack frame for `noconcat` and also breaks the symmetry a bit.  I think we should really make the `minErr` pass of the closure compiler smarter but that's a lot more work at this point in time.  I think that the current choice is an acceptable compromise for the present time.